app/assets/stylesheets/collection_management.scss

```diff
@@ -40,86 +52,13 @@
     border: 1px dashed #1385e5;
   }
 
-  .inner {
-    font-size: 14px;
-    color: black;
-    position: relative;
-    cursor: pointer;
-  }
-
-  .node {
-    display: inline-block;
-    width: 100%;
-    padding: 0px 5px;
-
-    .collection-label {
-      @extend .inner;
-      width: 400px;
-      height: 22px;
-      float: left;
-      margin-left: 15px;
-    }
-
-    .m-node .children {
-      transition: 1s;
-      font-size: 22px;
-    }
-
-    .collection-sync-info {
-      height: 22px;
-      float: left;
-      margin-left: 25px;
-    }
-
-    .root-label {
-      @extend .inner;
-      margin-left: 15px;
-      float: left;
-      padding-top: 4px;
-      padding-right: 0px;
-      height: 26px;
-      border-width: 0px;
-      width:94%;
-      box-shadow: 0px 0px 0px rgb(104, 104, 104);
-    }
-
-    .form-group {
-      margin-bottom: 0px;
-    }
-
-    &.is-active {
-      background-color: #D3D3D3;
-
-      .root-label{
-        background-color: #D3D3D3 
-      }
-    }
-  }
-
-  .root-actions {
-    float: right;
-    padding-top: 4px;
-  }
-
-  .actions {
-    float: right;
-    padding-top: 0px;
-  }
-
-  .collapse {
-    position: absolute;
-    left: 0;
-    cursor: pointer;
-    display: block;
-  }
-
   .caret-right:before {
-    content: '\25B8';
+    content: "\25B8";
     font-size: 15px;
   }
 
   .caret-down:before {
-    content: '\25BE';
+    content: "\25BE";
```

Comment:
I'm leaving this as is for now. There is a way to use fa, but these classes are used in different parts of the app and changing this would increase the scope of this PR. I'll do it in a separate commit/PR for all Tree views.